keps/sig-node/xxx-dynamic-resource-allocation/README.md

```diff
@@ -0,0 +1,1368 @@
+[supported limits].
+
+[supported limits]: https://git.k8s.io/community//sig-scalability/configs-and-limits/thresholds.md
+-->
+
+### Troubleshooting
+
+<!--
+This section must be completed when targeting beta to a release.
+
+The Troubleshooting section currently serves the `Playbook` role. We may consider
+splitting it into a dedicated `Playbook` document (potentially with some monitoring
+details). For now, we leave it here.
+-->
+
+###### How does this feature react if the API server and/or etcd is unavailable?
+
+###### What are other known failure modes?
+
+<!--
+For each of them, fill in the following information by copying the below template:
+  - [Failure mode brief description]
+    - Detection: How can it be detected via metrics? Stated another way:
+      how can an operator troubleshoot without logging into a master or worker node?
+    - Mitigations: What can be done to stop the bleeding, especially for already
+      running user workloads?
+    - Diagnostics: What are the useful log messages and their required logging
+      levels that could help debug the issue?
+      Not required until feature graduated to beta.
+    - Testing: Are there any tests for failure mode? If not, describe why.
+-->
+
+###### What steps should be taken if SLOs are not being met to determine the problem?
+
+## Implementation History
+
+<!--
+Major milestones in the lifecycle of a KEP should be tracked in this section.
+Major milestones might include:
+- the `Summary` and `Motivation` sections being merged, signaling SIG acceptance
+- the `Proposal` section being merged, signaling agreement on a proposed design
+- the date implementation started
+- the first Kubernetes release where an initial version of the KEP was available
+- the version of Kubernetes where the KEP graduated to general availability
+- when the KEP was retired or superseded
+-->
+
+## Drawbacks
+
+<!--
+Why should this KEP _not_ be implemented?
+-->
+
+## Alternatives
+
+<!--
+What other approaches did you consider, and why did you rule them out? These do
+not need to be as detailed as the proposal, but should include enough
+information to express the idea and why it was not acceptable.
+-->
+
+### ResourceClaimTemplate
+
+Instead of creating a ResourceClaim from an embedded template, the
+PodStatus could be extended to hold the same information as a
+ResourceClaimStatus. Every component which works with that information
+then needs permission and extra code to work with PodStatus. Creating
+an extra object seems simpler.
+
+### Reusing volume support as-is
+
+ResourceClaims are similar to PersistentVolumeClaims and also a lot of
+the associated logic is similar. An [early
+prototype](https://github.com/intel/proof-of-concept-cdi) used a
+custom CSI driver to manage resources.
+
+The user experience with that approach is poor because per-resource
+parameters must be stored in annotations of a PVC due to the lack of
+custom per-PVC parameters. Such parameters were [proposed
```

Comment:
would it be better to say "such annotations were..."?

Reply:
Changed into "Passing annotations as additional parameters was".